src/tests/intg/test_ldap.py

```diff
@@ -1890,7 +1884,7 @@ def test_rename_incomplete_group_rdn_changed(ldap_conn, rename_setup_cleanup):
 
     # The initgroups succeeds, but because saving the new group fails,
     # SSSD will revert to the cache contents and return what's in the cache
```

Comment:
IIUC this comment is no longer valid and should be removed or modified, correct?